docs/dev/testservers.rst

```diff
@@ -9,7 +9,7 @@ Test Server 1-3
 Deployment via Bamboo. Only for branches of the ``ls1intum/Artemis`` repository, no forks.
 Guide available in the Artemis Developer Confluence Space: `Deploying changes to test server`_.
 
-.. _`Deploying changes to test server`: https://confluence.ase.in.tum.de/display/Artemis/Deploying+changes+to+test+server
+.. _`Deploying changes to test server`: https://confluence.ase.in.tum.de/display/ArTEMiS/GitHub+Deployment
```

Comment:
Shouldn't we link this page instead [https://confluence.ase.in.tum.de/display/ArTEMiS/Bamboo+Deployment](https://confluence.ase.in.tum.de/display/ArTEMiS/Bamboo+Deployment)?

Reply:
in the slack dev channel it said that github is default workflow to deploy sth to a test server. so I assumed we should like the github confluence page

Reply:
i changed the description before the link to "deployment via github actions"